test/functional/wiki_controller_test.rb

```diff
@@ -572,4 +572,12 @@ def teardown
     get :show, id: @node.slug
     assert_response :success
   end
+
+  test "should render comment template when a comment=true parameter is passed in the get request" do
+    wiki = node(:wiki_page)
+    slug = wiki.path.gsub('/wiki/', '')
+    get "/wiki/data-logging?comments=true"
```

Comment:
@aspriya The action corresponding to the url doesn't exist. Have a look at thw `wiki_page` fixture again. Also note that the wiki path and slug is different in this case